.env.example

```diff
@@ -13,3 +13,7 @@ SENTRY_DSN="your-dsn"
 GITHUB_CLIENT_ID="MOCK_GITHUB_CLIENT_ID"
 GITHUB_CLIENT_SECRET="MOCK_GITHUB_CLIENT_SECRET"
 GITHUB_TOKEN="MOCK_GITHUB_TOKEN"
+
+# set this to true to prevent search engines from indexing the website
+# default to allow indexing for seo safety
+DISALLOW_INDEXING="false"
```

Comment:
I generally prefer variables not use negation. So this should instead be "ALLOW_INDEXING" which defaults to `true`